portfolio.html

```diff
@@ -165,7 +165,7 @@ <h1 class="about-me">
     <form action="https://formspree.io/f/xvodlvrj" method="POST">
       <input type="text" name="name" id="name" placeholder="Full name" class="input-name" required minlength="1" maxlength="30"> <br>
       <input type="email" id="email" placeholder="Email address" class="input-email" required minlength="1"><br>
-      <textarea name="text" minlength="1" required maxlength="500">Enter text here</textarea>
+      <textarea name="text" class="inputText" minlength="1" required maxlength="500">Enter text here</textarea>
       <button class="touch-button" type="submit">Get in touch</button>
     </form>
```

Comment:
- [ ] This components in this section are overlapping, the about me text is overlapping with the form, please kindly fix that. See images below. 

***The design***
![lockd2](https://user-images.githubusercontent.com/65852621/134505937-012566d4-e9d3-4733-90a6-670aa51f8dda.png)

***Your implementation**
![locki2](https://user-images.githubusercontent.com/65852621/134505994-f285741a-1818-4c4e-9b9e-188cf5a15b1c.png)
*